studio/remote_worker.py

```diff
@@ -42,12 +43,24 @@ def main(args=sys.argv):
     parsed_args, script_args = parser.parse_known_args(args)
     verbose = model.parse_verbosity(parsed_args.verbose)
     logger.setLevel(verbose)
+
+    config = None
+    if parsed_args.config is not None:
+        config = model.get_config(parsed_args.config)
+
     if parsed_args.queue.startswith('ec2_') or \
        parsed_args.queue.startswith('sqs_'):
         queue = SQSQueue(parsed_args.queue, verbose=verbose)
+    elif parsed_args.queue.startswith('rmq_'):
+        queue = RMQueue(
+            queue=parsed_args.queue,
+            route ='StudioML.*',
```

Comment:
what's route?

Reply:
amqp supports more message exchange patterns than the flat queuing inside the cloud vendor offerings.  route is used by amqp to allow a message switching and routing capability when the server gets a message.  I suppose it puts message selection and routing into the control of the message consumers environment.  I put the hard coded value in here to allow fixed selection to avoid getting into having to surface this to users so they can design their own switching and routing, doing that is probably going a little far as well.